src/main/java/uk/gov/ons/ctp/response/action/factory/ActionFeedbackFactory.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.ons.ctp.response.action.factory;
+
+import uk.gov.ons.ctp.response.action.message.feedback.ActionFeedback;
+import uk.gov.ons.ctp.response.action.message.feedback.Outcome;
+import uk.gov.ons.ctp.response.action.representation.Situation;
+
+public class ActionFeedbackFactory {
+  public static ActionFeedback create(String actionId, Situation situation, Outcome outcome) {
```

Comment:
Feels like overkill to have a factory to create an instance of ActionFeedback 🤔. Why not give the constructor this signature

Reply:
Oh that makes sense 👍. Forgot you mentioned that last week